src/plugins/ucx/ucx_backend.cpp

```diff
@@ -396,6 +408,9 @@ void nixlUcxEngine::progressThreadStart()
         // not enabled
         return;
     }
+    uint64_t counter;
+    // This resets eventfd internal counter to 0
+    std::ignore = read(pthrControlFd, &counter, sizeof(counter));
```

Comment:
To be on the safe side, maybe better to keep reading until it's done

Reply:
Nevermind, refactored control channel back to pipes.